packages/sdk/contractkit/src/wrappers/Election.ts

```diff
@@ -509,9 +510,15 @@ export class ElectionWrapper extends BaseWrapperForGoverning<Election> {
     epochNumber: number,
     useBlockNumber?: boolean
   ): Promise<GroupVoterReward[]> {
-    const blockchainParamsWrapper = await this.contracts.getBlockchainParameters()
-
-    const blockNumber = await blockchainParamsWrapper.getLastBlockNumberForEpoch(epochNumber)
+    let blockNumber: number
```

Comment:
be sure to add a patch changeset for contractkit about fixing ElectionWrapper.getGroupVoterRewards for l2